.github/BOTMETA.yml

```diff
@@ -335,9 +335,50 @@ files:
   $modules/network/icx/: sushma-alethea
   $modules/network/layer2/: $team_networking
   $modules/network/layer3/: $team_networking
+  $modules/network/meraki/meraki_admin.py: &meraki
```

Comment:
The `*meraki` means use the YAML anchor, which you defined on line 378


So you only need to list the files like this:
```
  $modules/network/meraki/meraki_admin.py: *meraki
  $modules/network/meraki/meraki_config_template.py: *meraki
```

Reply:
I made some changes which pulled some of the anchor references. Not sure if this is right, but please take a look. Thanks.